testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -423,18 +466,19 @@ def __init__(self, bindings, agent=None):
       agent: [SpinnakerAgent] The Spinnaker agent to bind to the scenario.
     """
     super(SpinnakerTestScenario, self).__init__(bindings, agent)
-    agent = self.agent
+    agent=self.agent
     self.__update_bindings_with_subsystem_configuration(agent)
     JournalLogger.begin_context('Configure Cloud Bindings')
     try:
       self.__init_google_bindings()
+      self.__init_azure_bindings()
```

Comment:
In head, you'll add AzureScenarioSupport to an array at the top of the file and not need to make this call.